slides/src/content/es6.md

```diff
@@ -24,67 +53,73 @@ class Toppings {
   }
   fetch(){
     getFromServer(function callback(){ 
-      this.formatToppings(); // not gonna work
+      this.formatToppings(); // not gonna work, because this will be undefined
     });
   }
 }
 
 var toppings = new Toppings();
 var lostThis = toppings.list;
 lostThis() // not gonna work
-var foundThis = toppings.list.bind(toppings);
-foundThis() // this will work
 ```
-
----
-
-## Arrow Functions
+- You can use call, bind or supply to pass `this` to change the context.
 
 ```js
-items.forEach(function(x) {
-  console.log(x);
-  incrementedItems.push(x+1);
-});
-
-items.forEach((x) => {
-  console.log(x);
-  incrementedItems.push(x+1);
-});
-
-incrementedItems = items.map((x) => { return x+1; });
-incrementedItems = items.map((x) => x+1); // for single return expression can simplify
-incrementedItems = items.map(x => x+1);   // when a single param can remove parantheses
+var foundThis = toppings.list.bind(toppings);
```

Comment:
I actually see a few `var` keywords in the slides. Since the slides are strictly about ES6, should we go ahead and replace them with `let` or `const`?